package.json

```diff
@@ -79,8 +79,9 @@
     "events": "^3.3.0",
     "fp-future": "^1.0.1",
     "parallax-js": "^3.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "postcss-copy": "^7.1.0",
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0",
```

Comment:
@2fd is the change in the `react` and `react-dom` versions intended? Also I notice that there are 33k additions to the `package-lock.json`, I suppose because it was migrated from `lockfileVersion: 1` to `lockfileVersion: 2` (npm 8 instrad of 6). I'm not against these changes at all, just wanted to make sure this will not have unintended consequences. I created a release candidate `decentraland-ui@3.53.2-rc` and tried it on the marketplace and it worked perfectly fine.

Reply:
no, is not intended, let me fix it